ci_watson/hst_helpers.py

```diff
@@ -140,3 +142,57 @@ def download_crds(refname, timeout=30, verbose=False):
 
     if verbose:
         print('Downloaded {} from {}'.format(refname, url))
+
+
+# Base definitions for HST tests
+class BaseHSTCal(BaseTest):
```

Comment:
I thought that we agree to leave test classes back over at the repo of code under test for now, no?